synapse/models/risk.py

```diff
@@ -942,6 +944,16 @@ def getModelDefs(self):
                         'doc': 'The currency in which payment was demanded.'}),
 
                 )),
+                ('risk:technique:masquerade', {}, (
+                    ('node', ('ndef', {}), {
```

Comment:
Is `ndef` pretty well understood? Maybe an `ex` on this prop?

Reply:
We can break out a `form` sub for these two ndef properties